nginx/templates/server.conf.template

```diff
@@ -1,5 +1,8 @@
+resolver 127.0.0.11 valid=2s; # 127.0.0.11 is from default /etc/resolv.conf
```

Comment:
2s is quite aggressive when everything is running. I think we can increase this to get more benefit from caching. Obviously this may make outages slightly longer but I don't think it will, because my assumption is...

1. API goes down
2. up to 2 seconds later the resolver cache expires
3. some time later API comes back up
4. the DNS resolves correctly

If API takes longer than 10s to start then there's no point having the resolve timeout less than that.

I'd go go for something between 10 and 30 seconds myself.

Reply:
Ok, cool, I'll bump it to 10.